app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,93 @@
+require 'chronic'
+
+class TasksController < ApplicationController
+  def index #homepage
+    @all_tasks = Task.all.order(date_completed: :desc)
+  end
+
+  def new
+    @task = Task.new
+    @people = Person.all
+    # @task = Task.find(params[:id])
+
+    render "new"
+  end
+
+  def create
+    @task = Task.new(tasks_create_params[:task])
```

Comment:
If you use `.create` here instead of `.new` then you won't have to call `.save` later